content/en/docs/setup/independent/high-availability.md

```diff
@@ -10,513 +10,217 @@ content_template: templates/task
-
-If possible, use a smart load balancing algorithm like "least connections", and use health checks so unhealthy nodes can be removed from circulation. Most providers will provide these features.
-{{% /tab %}}
-{{% tab name="On-Site" %}}
-In an on-site environment there may not be a physical load balancer available. Instead, a virtual IP pointing to a healthy master node can be used. There are a number of solutions for this including keepalived, Pacemaker and probably many others, some with and some without load balancing.
-
-As an example we outline a simple setup based on keepalived. Depending on environment and requirements people may prefer different solutions. The configuration shown here provides an _active/passive_ failover without load balancing. If required, load balancing can by added quite easily by setting up HAProxy, NGINX or similar on the master nodes (not covered in this guide).
-
-1. Install keepalived, e.g. using your distribution's package manager. The configuration shown here works with version `1.3.5` but is expected to work with may other versions. Make sure to have it enabled (chkconfig, systemd, ...) so that it starts automatically when the respective node comes up.
-
-2. Create the following configuration file _/etc/keepalived/keepalived.conf_ on all master nodes:
-     ```none
-     ! Configuration File for keepalived
-     global_defs {
-      router_id LVS_DEVEL
-     }
-
-     vrrp_script check_apiserver {
-      script "/etc/keepalived/check_apiserver.sh"
-      interval 3
-      weight -2
-      fall 10
-      rise 2
-     }
-
-     vrrp_instance VI_1 {
-        state <STATE>
-        interface <INTERFACE>
-        virtual_router_id 51
-        priority <PRIORITY>
-        authentication {
-            auth_type PASS
-            auth_pass 4be37dc3b4c90194d1600c483e10ad1d
-        }
-        virtual_ipaddress {
-            <VIRTUAL-IP>
-        }
-        track_script {
-            check_apiserver
-        }
-     }
-     ```
-
-    In the section `vrrp_instance VI_1`, change few lines depending on your setup:
-
-    * `state` is either `MASTER` (on the first master nodes) or `BACKUP` (the other master nodes).
-    * `interface` is the name of an existing public interface to bind the virtual IP to (usually the primary interface).
-    * `priority` should be higher for the first master node, e.g. 101, and lower for the others, e.g. 100.
-    * `auth_pass` use any random string here.
-    * `virtual_ipaddresses` should contain the virtual IP for the master nodes.
-
-3. Install the following health check script to _/etc/keepalived/check_apiserver.sh_ on all master nodes:
-     ```bash
-     #!/bin/sh
-
-     errorExit() {
-        echo "*** $*" 1>&2
-        exit 1
-     }
+At the end of this step, try `ssh`ing between nodes to ensure success.
```

Comment:
STEP! (are you getting tired of this yet? ;)):

1. To test, SSH between nodes.